brazilian-journal-of-experimental-design-data-analysis-and-inferential-statistics.csl

```diff
@@ -0,0 +1,492 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Brazilian Journal of Experimental Design, Data Analysis and Inferential Statistics</title>
+    <title-short>BJEDIS</title-short>
+    <id>http://www.zotero.org/styles/brazilian-journal-of-experimental-design-data-analysis-and-inferential-statistics</id>
+    <link href="http://www.zotero.org/styles/brazilian-journal-of-experimental-design-data-analysis-and-inferential-statistics" rel="self"/>
+    <link href="http://www.zotero.org/styles//associacao-brasileira-de-normas-tecnicas" rel="template"/>
+    <link href="https://bjedis.org/author-1" rel="documentation"/>
+    <author>
+      <name>Fernando Gomes</name>
+      <email>fgsj@ufrj.br</email>
+    </author>
+    <category citation-format="numeric"/>
+    <summary>Style based on ABNT numeric, V1.1</summary>
+    <updated>2021-02-09T15:04:30+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="no date">[no date]</term>
+      <term name="in">in</term>
+      <term name="online">online</term>
+      <term name="accessed">accessed</term>
+      <term name="retrieved">Available</term>
+      <term name="from">from</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" name-as-sort-order="all">
```

Comment:
`et-al-use-first` should always be at least one lower the `et-al-min` -- it doesn't make sense to have et al with 19 authors but then also print the first 19 authors followed by et al.
We generally prefer to have et al in the citation or bibliography section where possible.

Reply:
Thank you so much. We've corrected it.
<name and="text" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="18" name-as-sort-order="all">
Our publications can receive up to 19 authors. So we would like to keep this number.